server/services/CycleService.js

```diff
@@ -294,9 +310,171 @@ async function getAllTestVersions() {
     }
 }
 
+/**
+ * Saves a test result and marks the test as "complete"
+ *
+ * @param {object} result - result object to save
+ * @return {object} - the saved result object (the same data, now with "id" and "status: complete")
+ *
+ * @example
+ *
+ *     result = {
+ *       test_id,
+ *       run_id,
+ *       user_id,
+ *       result
+ *     };
+ */
+async function saveTestResults(testResult) {
+    try {
+        let statusId = (
+            await sequelize.query(`
+             SELECT
+               id
+             FROM
+               test_status
+             WHERE
+               test_status.name = 'complete'
+        `)
+        )[0][0].id;
```

Comment:
`[0][0].id` seems brittle.

Reply:
As we discussed, I have a backlog task in asana to switch all this code to sequalize